azure-eventhubs/dev_requirements.txt

```diff
@@ -0,0 +1,9 @@
+pytest>=3.4.1
+pytest-asyncio>=0.8.0; python_version > '3.4'
+azure-servicebus==0.50.0
```

Comment:
Should be relative installation

Reply:
Unless the dev_setup is now able to get that it has to install azure-servicebus first, to be checked with Scott K I'm not sure